pypit/arutils.py

```diff
@@ -417,7 +422,7 @@ def get_splknots(xarr, yarr, num, minv=None, maxv=None, maxknots=None):
     drv = np.ceil(drv).astype(np.int)
     drv[np.where(drv<2)] = 2
     if maxknots is not None: drv[np.where(drv>maxknots)] = maxknots
-    print drv
+    print(drv)
```

Comment:
I think this line can just be deleted, right? If not, we should probably use the msgs routine for printout.